src/xrCore/Threading/TaskManager.cpp

```diff
@@ -88,7 +90,7 @@ class FallbackTaskAllocator
 class TaskAllocator
 {
     size_t m_allocated{};
-    Task   m_storage[TASK_STORAGE_SIZE];
+    Task   m_storage[TASK_STORAGE_SIZE]{};
```

Comment:
This is left uninitialized on purpose – to reduce construction time.
In fact, this storage is a ring buffer with `Task` destructors never being called on it, so it's guaranteed to have garbage.
Hence, no default initialization is ever needed.
```suggestion
    Task   m_storage[TASK_STORAGE_SIZE];
```